src/rules.c

```diff
@@ -53,6 +53,22 @@ void SchemaRuleArgs_Free(SchemaRuleArgs *rule_args) {
   rm_free(rule_args);
 }
 
+void SchemaRulesArgs_Free(RedisModuleCtx *ctx) {
```

Comment:
You can use `RSDummyContext`

Reply:
I wanted to use the context from one of the original calls. It gives better logs.

Reply:
But we can still use dummy instead of NULL, right?